src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -61,15 +65,18 @@ const RunDataset = ({
             size="large"
           >
             {Object.keys(data)
-              .sort()
+              .sort((a, b) => {
+                return a.localeCompare(b);
+              })
```

Comment:
Out of curiosity, I wonder if there is any specific reasons why you prefer to use `localeCompare` other than the normal `sort` method?